ci-operator/step-registry/workers-rhel/gcp-provision/workers-rhel-gcp-provision-commands.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+REGION="${LEASED_RESOURCE}"
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export SSH_PUB_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-publickey
+
+infra_id=$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)
+
+VPC_CONFIG="${SHARED_DIR}/customer_vpc_subnets.yaml"
+NETWORK=$(/tmp/yq r "${VPC_CONFIG}" 'platform.gcp.network')
+COMPUTE_SUBNET=$(/tmp/yq r "${VPC_CONFIG}" 'platform.gcp.computeSubnet')
+if [[ -z "${NETWORK}" || -z "${COMPUTE_SUBNET}" ]]; then
+  echo "Could not find VPC network or compute subnet." && exit 1
+fi
+mapfile -t avail_zones < <(gcloud compute regions describe ${REGION} --format=json | jq -r .zones[] | cut -d "/" -f9)
+
+# Start to provision rhel instances in existing VPC
+for count in $(seq 1 ${RHEL_WORKER_COUNT}); do
+  echo "$(date -u --rfc-3339=seconds) - Provision ${infra_id}-rhel-${count} ..."
+  gcloud compute instances create "${infra_id}-rhel-${count}" \
```

Comment:
Do we need deprovision ref to destroy these rhel workers, or the installer destroyer can find out these workers because of infra_id tag?

Reply:
Yes, installer would find them and delete them during destroying the cluster, see [build log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/29550/rehearse-29550-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.10-installer-rehearse-gcp/1539152436336267264/artifacts/installer-rehearse-gcp/ipi-deprovision-deprovision/build-log.txt).